src/routers/StackRouter.js

```diff
@@ -186,7 +186,11 @@ export default (
       }
 
       // Handle navigation to other child routers that are not yet pushed
-      if (action.type === NavigationActions.NAVIGATE) {
+      if (
+        action.type === NavigationActions.NAVIGATE &&
+        action.routeName !== 'DrawerOpen' &&
```

Comment:
I don't want to hard-code route names in the router. Especially since these names are random names that drawer navigator chose to use and they don't have any special meaning.